lib/utils.ts

```diff
@@ -1,4 +1,4 @@
-import * as Bchaddr from "bchaddrjs-slp";
+import * as Bchaddr from "bchaddrjs-slp-2";
```

Comment:
Why is this library being used? Afaict there isn't any difference other than the -2 version is behind by 1 commit.